### ISSUE TITLE
Error creating container

### ISSUE THREAD

**Issue by vinda-parab** [QAInsights/jmeter-docker-extension#13]
I pulled the image - for AMD64
<img width="679" alt="image" src="https://github.com/QAInsights/jmeter-docker-extension/assets/67003742/c82448ed-71eb-49c5-8762-14c9abd699b5">

However when i run the test plan - Getting below error.

Error creating container WARNING: the requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64/v3) and no specific platform was requested 

@QAInsights Can you please help?

**Comment by QAInsights** [comment]
@vinda-parab Did you try running it from the Docker Desktop UI?

**Comment by vinda-parab** [comment]
> @vinda-parab Did you try running it from the Docker Desktop UI?

Hi @QAInsights  Thanks for checking.
Yes, this is on Docker Desktop. I got the issue. The issue was with the jmeter image. It supports only ARM64.
To resolve this i had to use a different JMeter image. With this its executing the tests now. But getting a below issue to generate the results. Can you pls check and help?

<img width="940" alt="image" src="https://github.com/QAInsights/jmeter-docker-extension/assets/67003742/178759bb-1e01-4e59-ad85-0c0ac316fea5">


And here are the logs:
<img width="941" alt="image" src="https://github.com/QAInsights/jmeter-docker-extension/assets/67003742/c1d55d7c-dab4-44d4-a057-0c0300dd31b6">

For every run - it tries to find the different file in the container.
<img width="388" alt="image" src="https://github.com/QAInsights/jmeter-docker-extension/assets/67003742/743a1da1-201a-4981-abfe-96c35b12c977">

**Comment by QAInsights** [comment]
Yes, the image is compatible with ARM64. Let me see to add support for multiple architectures.

**Comment by vinda-parab** [comment]
> Yes, the image is compatible with ARM64. Let me see to add support for multiple architectures.

How to resolve the issue - mentioned in above comment? The reports are not getting saved. Please check.

**Comment by QAInsights** [comment]
@vinda-parab You can't use others' images. You must use `qainsights\jmeter` to get your reports. Let me rebuild the image and publish it.

**Comment by vinda-parab** [comment]
> @vinda-parab You can't use others' images. You must use `qainsights\jmeter` to get your reports. Let me rebuild the image and publish it.

@QAInsights Thank you for taking this up. Will keep checking here for your update. We are excited to use this extension in our project.

**Comment by vinda-parab** [comment]
@QAInsights If possible can you please let me know by when will you be able to publish the required image?
@QAInsights Any update?
